### ISSUE TITLE
ProDOS 2.4.2 BUG FIX for issue preventing A2osX 0.94 from starting

### ISSUE THREAD

**Issue by bobbimanners** [ProDOS-8/ProDOS8-Testing#55]
## Description
There is a problem in ProDOS 2.4.2 which prevents `A2OSX.SYSTEM` from launching correctly.  This affects the most recent test version of A2osX, version 0.94.

The A2osX project has identified the issue and patched `PRODOS`.  This fix has been verified to correct the problem.  It would be good to upstream it.

## Expected Behavior
`A2OSX.SYSTEM` should launch.

## Actual Behavior
`A2OSX.SYSTEM` crashes during launch.

## Possible Fix
I will attach a screenshot of Slack chat with the details of the fix.

## Steps to Reproduce
1. Download a recent test release of A2osX (v0.94).
2. Boot ProDOS 2.4.2
3. Try launching `/FULLBOOT/A2OSX.SYSTEM`.

## Context
Trying to run the awesome A2osX :)

## Your Environment
ProDOS 2.4.2

### Hardware:
Enhanced Apple //e
Ultrawarp
GW 8MB Aux slot RAM
Uthernet-II
MicroDrive/Turbo

### ProDOS8 version:
2.4.2

### Non-Operating System release software being used: 
A2osX 0.94 test release.

**Comment by bobbimanners** [comment]
Details of fix.

![Screenshot from 2021-03-18 17-27-57](https://user-images.githubusercontent.com/11758908/111700319-9cb61e00-880f-11eb-9ce3-a20ad4e07702.png)
![Screenshot from 2021-03-18 17-28-03](https://user-images.githubusercontent.com/11758908/111700355-a8094980-880f-11eb-95e6-c9ca2ae6a4d6.png)

**Comment by bobbimanners** [comment]
The `PRODOS` system file on this disk image has the patch applied:
[ADTPRO-2.1.0.dsk.zip](https://github.com/ProDOS-8/ProDOS8-Testing/files/6167326/ADTPRO-2.1.0.dsk.zip)
```
ADTPRO-2.1.0.dsk /ADTPRO.2.1.0/
PRODOS SYS 17,128 Locked
BASIC SYS 10,240 Locked
STARTUP.SYSTEM SYS 1,034
ADTPRO SYS 348
ADTPROAUD SYS 348
ADTPROETH SYS 348
ADTPRO.BIN BIN 14,205
ADTPROAUD.BIN BIN 17,008
ADTPROETH.BIN BIN 19,205
ProDOS format; 55,296 bytes free; 88,064 bytes used.
```

**Comment by JohnMBrooks** [comment]
Fixed in 2.4.3

**Comment by wyatt-wong** [comment]
Would it be possible to reuse address $BF4B and $BF4C instead of just replaced them by 2 NOPs ?
